diff --git a/scripts/generator.py b/scripts/generator.py
index d6f35b0..fd7d774 100644
--- a/scripts/generator.py
+++ b/scripts/generator.py
@@ -63,8 +63,6 @@ def main():
     es_template.generate(nested, ecs_generated_version, out_dir, args.mapping_settings)
     es_template.generate_legacy(flat, ecs_generated_version, out_dir, args.template_settings, args.mapping_settings)
     beats.generate(nested, ecs_generated_version, out_dir)
-    if args.include or args.subset or args.exclude:
-        exit()
 
     ecs_helpers.make_dirs(docs_dir)
     asciidoc_fields.generate(nested, ecs_generated_version, docs_dir)
